File: drift/src/main/java/drift/com/drift/model/PreMessage.java
package drift.com.drift.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by eoin on 22/02/2018.
 */

public class PreMessage {

    @Expose
    @SerializedName("body")
    public String messageBody;
    @Expose
    @SerializedName("sender")
    public User sender;
}

File: drift/src/main/java/drift/com/drift/model/PreMessage.kt
package drift.com.drift.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by eoin on 22/02/2018.
 */

class PreMessage {

    @Expose
    @SerializedName("body")
    var messageBody: String? = null
    @Expose
    @SerializedName("sender")
    var sender: User? = null
}
